docs/runtimes/application.ipynb

```diff
@@ -92,6 +92,74 @@
    "metadata": {
```

Comment:
<p>Specifying "typically none, unless" emphasises open source over enterprise which IMO is wrong. I would say something like "is none for open source and access-key for Iguazio..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>